static/src/stylesheets/module/commercial/gustyle/_gu-comcontent.scss

```diff
@@ -35,6 +35,7 @@
     color: #ffffff;
 }
 
+.content--immersive .from-content-api .gu-display__content-title,
 .gu-display__content-title {
     @include f-headlineSans;
```

Comment:
Would using `!important` with a note be more explicit and easier to fathom for someone seeing this later?